sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/list_keys_responses.hpp

```diff
@@ -26,10 +26,11 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
    * @brief Define a single page to list the keys from the Key Vault.
    *
    */
-  class KeyPropertiesPageResult final : public Azure::Core::PagedResponse<KeyPropertiesPageResult> {
+  class KeyPropertiesPageResponse final
```

Comment:
Call these type renames out in the changelog.

Reply:
```suggestion
  class KeyPropertiesPagedResponse final
```

Reply:
It's `PagedResponse`, with the (d), since the type is called `PagedResponse<T>`. Make the same change elsewhere.

This will make it consistent with other existing types like:
https://github.com/Azure/azure-sdk-for-cpp/blob/07861a9687449ec360299a90a0b824c0eeab0903/sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_container_client.hpp#L211-L213